src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -891,7 +891,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     {
         const auto modifiers{ _GetPressedModifierKeys() };
         auto handled = false;
-        if (vkey == VK_MENU && !down)
+        if (vkey == VK_MENU && !down && !_isReadOnly)

```

Comment:
Should we add this check to the `F7` handler below too? Maybe just the whole method?